radiation/python/radlw/lwrad_gt4py.py

```diff
@@ -0,0 +1,1800 @@
+
+    if var in ["semis", "icsdlw", "tsfg", "de_lgth"]:
+        indict[var] = np.tile(tmp[:, None, None], (1, 1, nlp1))
+    elif var == "faerlw":
+        tmp2 = np.insert(tmp, 0, 0, axis=1)
+        indict[var] = np.tile(tmp2[:, None, :, :, :], (1, 1, 1, 1, 1))
+    elif var == "gasvmr" or var == "clouds":
+        tmp2 = np.insert(tmp, 0, 0, axis=1)
+        indict[var] = np.tile(tmp2[:, None, :, :], (1, 1, 1, 1))
+    elif var in ["plyr", "tlyr", "qlyr", "olyr", "dz", "delp"]:
+        tmp2 = np.insert(tmp, 0, 0, axis=1)
+        indict[var] = np.tile(tmp2[:, None, :], (1, 1, 1))
+    elif var in ["plvl", "tlvl"]:
+        indict[var] = np.tile(tmp[:, None, :], (1, 1, 1))
+    elif var[-3:] == "tbl":
+        indict[var] = np.tile(tmp[None, None, None, :], (npts, 1, nlp1, 1))
+    else:
+        indict[var] = tmp[0]
+
+indict_gt4py = dict()
+
+for var in invars:
+    if var == "faerlw":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlp1, (DTYPE_FLT, (nbands, 3))
+        )
+    elif var == "gasvmr" or var == "clouds":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlp1, (DTYPE_FLT, (indict[var].shape[3],))
+        )
+    elif var == "icsdlw":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlp1, DTYPE_INT
+        )
+    elif var[-3:] == "tbl":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlp1, type_ntbmx
+        )
+    elif indict[var].size > 1:
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlp1, DTYPE_FLT
+        )
+    else:
+        indict_gt4py[var] = indict[var]
+
+outdict_gt4py = dict()
+
+for var in outvars:
+    if var in ["htlwc", "htlw0", "cldtaulw"]:
+        outdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_FLT)
+    else:
+        outdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_FLT)
+
+locdict_gt4py = dict()
+
+for var in locvars:
+    if var == "rfrate":
+        locdict_gt4py[var] = create_storage_zeros(
+            backend, shape_nlp1, (DTYPE_FLT, (nrates, 2))
+        )
+    elif var == "wx":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_maxxsec)
+    elif var == "pwvcm" or var == "tem00":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_FLT)
+    elif var == "lcf1":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_BOOL)
+    elif var == "colamt":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_maxgas)
+    elif var in [
+        "semiss",
+        "secdiff",
+        "pklay",
+        "pklev",
+        "htrb",
+    ]:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_nbands)
+    elif var == "semiss0":
+        locdict_gt4py[var] = create_storage_ones(backend, shape_nlp1, type_nbands)
```

Comment:
wow a variable is initialized as 1s? is this a boolean?

Reply:
This is set to ones in the fortran. Wasn't sure if it was better to do it like this or in the stencil, since it is a local variable so it will just be created as a storage every time